test/components/HomeSpec.js

```diff
@@ -1,44 +1,32 @@
 import test from 'tape'
 import React from 'react'
 import { MemoryRouter } from 'react-router-dom'
-import {
-  mount,
-  // render,
-  shallow
-} from 'enzyme'
+import { Provider } from 'react-redux'
+import { mount } from 'enzyme'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'enzyme'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->